planner/core/exhaust_physical_plans.go

```diff
@@ -2194,9 +2194,19 @@ func (ls *LogicalSort) getNominalSort(reqProp *property.PhysicalProperty) *Nomin
 func (ls *LogicalSort) exhaustPhysicalPlans(prop *property.PhysicalProperty) ([]PhysicalPlan, bool) {
 	if MatchItems(prop, ls.ByItems) {
 		ret := make([]PhysicalPlan, 0, 2)
-		ret = append(ret, ls.getPhysicalSort(prop))
+		ps := ls.getPhysicalSort(prop)
+		ps.doInlineProjection = ls.doInlineProjection
```

Comment:
Suggest don't add `doInlineProjection` member to `logicalSchemaProducer` & `physicalSchemaProducer`, as we don't have a common place to assign `doInlineProjection` from logical to physical, and have little improvement to performance, but increase codes complexity.

Reply:
Indeed, if `doInlineProjection` does not handle a special condition logic.